service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -108,6 +110,15 @@ private void run1218(CaseDetails caseDetails) {
         removeSpecificPlacements(caseDetails, placementToRemove);
     }
 
+    private void run1243(CaseDetails caseDetails) {
+        var migrationId = "DFPL-1243";
+        var possibleCaseIds = List.of(1675697653441050L);
+
+        migrateCaseService.doCaseIdCheckList(caseDetails.getId(), possibleCaseIds, migrationId);
+
+        caseDetails.getData().remove(POSITION_STATEMENT_LIST_CHILD);
```

Comment:
Are we removing the whole list of position statement child?
FYI, there is a fnuction in `MigrateCaesService.removePositionStatementChild` for removing position statement child.